contracts/BitfieldComparison.sol

```diff
@@ -0,0 +1,222 @@
+// "SPDX-License-Identifier: Apache-2.0"
```

Comment:
Exactly, comparison was the purpose. Since this repository is for testing and experimentation, I'd leave the comparison files in there. Added a comment, but would it leave in contracts, since testing a contract is much easier and simpler to understand than testing a lib.